RSFormView/Classes/ViewModels/FormViewModel.swift

```diff
@@ -20,6 +20,27 @@ public enum FieldType: Equatable {
   case expiration
   case password
   case picker
+  //if you're adding a new type please update the == function, otherwise bad stuff is going to happen
```

Comment:
maybe explaining the 'bad stuff' would be better. WDYT?

Reply:
I don't even know what may happen haha
jk, I'll redact better